pkg/metadata/manager.go

```diff
@@ -0,0 +1,374 @@
+package metadata
+
```

Comment:
Does it still make sense to put this in `pkg/metadata/`? Maybe it should belong under `pkg/cgroups/` instead?

Reply:
Moved it under `pkg/cgroups/manager` as you had originally suggested.